session/session.go

```diff
@@ -331,7 +332,11 @@ func (manager *Manager) sessionID() (string, error) {
 	if n != len(b) || err != nil {
 		return "", fmt.Errorf("Could not successfully read from the system CSPRNG")
 	}
-	return hex.EncodeToString(b), nil
+	sid := hex.EncodeToString(b)
+	if manager.config.SessionIDPrefix != "" {
+		sid = manager.config.SessionIDPrefix + sid
+	}
+	return sid, nil
```

Comment:
Actually you don't need the `if else`, because if `manager.config.SessionIDPrefix` is empty return `manager.config.SessionIDPrefix + hex.EncodeToString(b)` is same as `hex.EncodeToString(b)`

Reply:
Wow